var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -90,8 +91,8 @@ class Seacas(CMakePackage):
     depends_on('adios2@develop~mpi', when='+adios2 ~mpi')
     depends_on('adios2@develop+mpi', when='+adios2 +mpi')
     depends_on('matio', when='+matio')
-    depends_on('metis+int64+real64', when='+metis ~mpi')
-    depends_on('parmetis+int64+real64', when='+metis +mpi')
+    depends_on('metis+int64+real64', when='+metis')
+    depends_on('parmetis+int64', when='+metis +mpi')
```

Comment:
```suggestion
    with when('+metis'):
        depends_on('metis+int64+real64')
        depends_on('parmetis+int64', when='+mpi')
```

could be a bit nicer to follow